cmd/server/main.go

```diff
@@ -57,7 +59,12 @@ func main() {
 
 	if *logFormatJSON {
 		jsonFactory := json.Factory{}
-		logger, _ := jsonFactory.Create(logsapi.LoggingConfiguration{Format: "json"})
+		logger, _ := jsonFactory.Create(
```

Comment:
Lets open a follow up to migrate this to `mlog`: https://github.com/Azure/azure-workload-identity/pull/707

Reply:
Opened https://github.com/Azure/kubernetes-kms/issues/236 to track it